.github/workflows/build.yml

```diff
@@ -0,0 +1,93 @@
+name: Continuous Integration Checks
```

Comment:
Not a dealbreaker but, wondering if there's a way to get the colors that Travis has, in github actions output? They're nice

Reply:
> How do we make this run? The "Checks" tab shows no checks here? I dont see any options in the repo settings to turn it on, or does it need to be on master first?

As @valentinewallace mentioned, because I am not a collaborator and I am using a fork, the new workflow I bring in does not get triggered on rust-bitcoin/rust-lightning repo. This is to protect any `secrets` you may have setup for GitHub Action on this repo, stopping non-maintainers to reveal them by echo-ing them with new workflow/PRs.

However, the actions are triggered on my fork, so you can see the result at https://github.com/D4nte/rust-lightning/actions for the same branch.

Once this is in master, I expect the checks to be triggered for any new PR, fork or not.

Reply:
Sorted :)